src/Core/src/Platform/Android/StrokeExtensions.cs

```diff
@@ -5,6 +5,26 @@ namespace Microsoft.Maui.Platform
 {
 	public static class StrokeExtensions
 	{
+		public static void UpdateBorderStroke(this AView platformView, IBorderStroke border)
```

Comment:
Do Windows require changes too?

Reply:
Maybe? Was traveling without a windows machine. Was going to check tomorrow.

Reply:
Yes,  taking a look to [WrapperView ](https://github.com/dotnet/maui/blob/b09979d93ac544d4b0dd9fd0df5c25da7e91392c/src/Core/src/Platform/Windows/WrapperView.cs) on Windows the logic to manage the border is missing. I think we should use the `BorderChanged` method to detect changes and `ContentPanel `control instead Grid in the WrapperView.

Reply:
I will send another PR with the Windows implementation. In that way we can merge and not block this PR.